### PR TITLE
adds links to docs and applies formatting rules

### PR DESCRIPTION
**Pull request by ksatirli**
Hey @stmcallister,

this PR adds some documentation to make the samples clearer and splits code out into a format that is more commonly expected.

Hope this helps!

### CODE REVIEW

File: main.tf [hunk 1]
```diff
@@ -62,8 +62,9 @@ resource "pagerduty_escalation_policy" "ep" {
   }
 }
 
-# /* SCHEDULE */
-resource "pagerduty_schedule" "foo" {
+# SCHEDULE
+# see https://registry.terraform.io/providers/PagerDuty/pagerduty/latest/docs/resources/schedule
+resource "pagerduty_schedule" "checkout_service" {
```
Comment (stmcallister) [inline main.tf:67]: Interesting. So, is it recommended that we use the naming convention of `checkout_service` for all the objects related to the `checkout_service`? So the schedule would have this name, as well as the escalation policy above?

**Review by stmcallister — COMMENTED** [review]
Thank you for this @ksatirli ! I just have a question before approving.

File: main.tf (hunk 1 above)
Comment (ksatirli) [reply main.tf:67]: We generally go with one of two names:

* `main` in a module context (also commonly seen as `this`, but that's less clear imo)

* the shortest, most logical reference

In this case, the resources related to `checkout_service` have that as common (= shared) name

**Review by stmcallister — APPROVED** [review]
Cool! Thanks for the context! 👍 🌮 🎉

**Pull request merged**